.changeset/config.json

```
@@ -0,0 +1,11 @@
+{
+    "$schema": "https://unpkg.com/@changesets/config@2.2.0/schema.json",
+    "changelog": false,
+    "commit": false,
+    "fixed": [],
+    "linked": [],
+    "access": "public",
+    "baseBranch": "main",
+    "updateInternalDependencies": "patch",
+    "ignore": []
```

These settings only apply to monorepos, see: https://github.com/changesets/changesets/blob/main/docs/config-file-options.md
```suggestion
    "access": "public",
    "baseBranch": "main"
```